src/buildEjectCommand/index.js

```diff
@@ -382,6 +382,9 @@ function buildEjectCommand(
 
       filestat = os.stat(filename)
 
+      # TODO: we probably should handle symlinks too in a special way,
```

Comment:
Not yet. Though with pnpm node_modules is a symphony no farm and things seems to work. We need to a proper test case though.